src/site/_data/paths/animations.json

```diff
@@ -0,0 +1,43 @@
+{
+  "slug": "animations",
+  "cover": "/images/collections/animations.svg",
+  "title": "Animations",
+  "updated": "September 25, 2020",
+  "description": "Techniques for developing high-performance CSS animations.",
+  "overview": "To do.",
+  "topics": [
+    {
+      "title": "Concepts",
+      "pathItems": [
+        "animations-overview",
+        "animations-guide"
+      ]
+    },
+    {
+      "title": "Guides",
+      "pathItems": [
+        {
+          "title": "Animation syntax basics",
+          "url": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations"
+        },
```

Comment:
@robdodson this change requires `pathItems` to either accept a string or an object with the properties `title` and `URL`